src/spatial/math.js

```diff
@@ -598,12 +602,14 @@ Crafty.math.Vector2D = (function () {
      * @kind Method
      * 
      * Scales this vector by the passed amount(s)
-     * If scalarY is omitted, scalarX is used for both axes
      *
      * @public
-     * @sign public {Vector2D} scale(Number[, Number]);
-     * @param {Number} scalarX
-     * @param {Number} [scalarY]
+     * @sign public {Vector2D} scale(Number scale);
+     * @param {Number} scale - The amount to scale by
+     * 
+     * @sign public {Vector2D} scale(Number scalarX, Number scalarY);
```

Comment:
I documented this one as having two signatures, rather than a single signature with an optional param -- my rationale was that the first parameter has different meanings in those two cases.

I'm not sure I was consistent in that approach everywhere in this file.